elasticsearch5/src/main/java/org/vertexium/elasticsearch5/ElasticsearchSearchQueryBase.java

```diff
@@ -513,6 +525,43 @@ protected IdStrategy getIdStrategy() {
         return createIterable(response, filterParameters, hitsIterable, response.getTookInMillis(), hits);
     }
 
+    private List<ElasticsearchVertex> getElasticsearchVertices(SearchHits hits, FetchHints fetchHints, Authorizations authorizations) {
+        return StreamUtils.stream(hits)
+                .map(hit -> {
+                    String elementId = hit.getField(Elasticsearch5SearchIndex.ELEMENT_ID_FIELD_NAME).getValue();
+                    return new ElasticsearchVertex(
+                            getGraph(),
+                            elementId,
+                            fetchHints,
+                            authorizations
+                    );
+                }).collect(Collectors.toList());
+    }
+
+    private List<ElasticsearchEdge> getElasticsearchEdges(SearchHits hits, FetchHints fetchHints, Authorizations authorizations) {
+        return StreamUtils.stream(hits)
+                .map(hit -> {
+                    String inVertexId = null;
+                    String outVertexId = null;
+                    String label = null;
+                    if (fetchHints.equals(FetchHints.EDGE_REFS)) {
```

Comment:
Shouldn't need this check. `FetchHints.EDGE_REFS` should only be applied to vertices. And won't in, out, and label be in the search result anyway.